src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -156,6 +170,11 @@
             <v-divider class="mt-7 mb-5"></v-divider>
           </template>
 
+          <template v-else-if="filing.isCompleteIA && tempRegNumber">
```

Comment:
Why do you need both conditions? Isn't isCompleteIA enough?

Reply:
Based on what I see on the api, when we have a real business number the complete IA should return as part of the history, but in that case we don't need to show this piece.

Reply:
Ah, you are correct!